trove/derive/osfmap_json_mini.py

```diff
@@ -0,0 +1,61 @@
+from trove.vocab import namespaces as ns
+from trove.derive.osfmap_json import OsfmapJsonFullDeriver
+from trove.vocab.namespaces import TROVE
+
+INCLUDED_PREDICATE_SET = frozenset({
+    ns.RDF.type,
+    ns.DCTERMS.title,
+    ns.DCTERMS.creator,
+    ns.DCTERMS.date,
+    ns.DCTERMS.created,
+    ns.FOAF.name,
+    ns.OWL.sameAs,
+    ns.DCTERMS.conformsTo,
+    ns.DCTERMS.dateCopyrighted,
+    ns.DCTERMS.description,
+    ns.DCTERMS.hasPart,
+    ns.DCTERMS.isVersionOf,
+    ns.DCTERMS.modified,
+    ns.DCTERMS.publisher,
+    ns.DCTERMS.rights,
+    ns.DCTERMS.subject,
+    ns.DCTERMS.isPartOf,
+    ns.DCTERMS.identifier,
+    ns.SKOS.inScheme,
+    ns.SKOS.prefLabel,
+    ns.OSFMAP.affiliation,
+    ns.OSFMAP.archivedAt,
+    ns.DCTERMS.dateAccepted,
+    ns.DCTERMS.dateModified,
+    ns.OSFMAP.hostingInstitution,
+    ns.OSFMAP.keyword,
+    ns.OSFMAP.contains,
+    ns.OSFMAP.fileName,
+    ns.OSFMAP.filePath,
+    ns.OSFMAP.isContainedBy
+})
+
+
+class IndexcardJsonDeriver(OsfmapJsonFullDeriver):
```

Comment:
(was going to ignore this but then i got confused often enough while reading it seems worthwhile to) rename `IndexcardJsonDeriver` to `OsfmapJsonMiniDeriver`, to be consistent with the module and parent class naming